### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/how-to-render-a-visual-style-element.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Render a Visual Style Element"
+description: Learn how to render a visual style element in Windows Forms by means of a 2-step procedure.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-render-a-visual-style-element.md:3]: Is there a way to increase the number of characters to over 100?

File: dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How To: Opt Out of File Dialog Box Automatic Upgrade"
+description: Learn how to opt out of a file dialog box automatic upgrade in Windows Forms in this 1-step procedure.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md:3]: I think this needs punctuation...

"Learn how to opt out of a file dialog box, automatic upgrade in Windows Forms in this 1-step procedure."

Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md:3]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How To: Opt Out of File Dialog Box Automatic Upgrade"
+description: Learn how to opt out of a file dialog box, automatic upgrade in Windows Forms in this 1-step procedure.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md:3]:
```suggestion
description: Learn how to revert the default file dialog box appearance to a Windows XP appearance.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**